modules/custom/express_content_edit_lock/express_content_edit_lock.module

```diff
@@ -247,10 +247,10 @@ function _express_content_edit_lock_recursive(&$element) {
 function express_content_edit_lock_user_login(&$edit, $account) {
   if (module_exists('cu_users')) {
     $devs = cu_users_get_users('developer');
```

Comment:
I think if it is users of the developer role, then you should just load that role with the users rather than only get the ones hardcoded in the cu_users module.

Example:
```php
function express_users_by_role($role_name) {
  $role = user_role_load_by_name($role_name);
  $query = 'SELECT ur.uid
    FROM {users_roles} AS ur
    WHERE ur.rid = :rid';
  $result = db_query($query, array(':rid' => $role->rid));
  $uids = $result->fetchCol();
  return user_load_multiple($uids);
}

// Then in express_content_edit_lock_user_login().
    $devs = express_users_by_role('developer');
    $dev_usernames = array_map(function ($val){
      return $val->name;
    }, $devs);
```

Reply:
Word from on high is to keep the hardcoded list.